src/test/groovy/org/osiam/test/integration/ChangeEmailIT.groovy

```diff
@@ -184,6 +203,17 @@ class ChangeEmailIT extends AbstractIT {
                 temp = it.getValue()
         }
         temp == newEmailValue
+
+        //Waiting at least 5 seconds to ensure that an email was send
+        Thread.sleep(5000)
```

Comment:
I know this is a bad idea to use `Thread.sleep` in a test... should I use [CountDownLatch](https://docs.oracle.com/javase/7/docs/api/java/util/concurrent/CountDownLatch.html) instead? Wait until one email was received by greenmail, right?